### PR TITLE
Bug 1343200 : added section on multihost requirements to prerequisites

### PR DESCRIPTION
**Pull request by bfallonf**
Added a small para on what's needed for multihost requirements and changed the section name to be more inclusive.

I'll ask for an ack in the BZ.

### CODE REVIEW

**Comment by bfallonf** [comment]
@rjhowe I tagged you in the BZ, but not here.

So...bump.

**Comment by rjhowe** [comment]
LGTM

**Comment by bfallonf** [comment]
Thanks @rjhowe I'll go forward with this then.

@tpoitras Can I get you to give this a read-over? Thanks!

File: install_config/install/prerequisites.adoc [hunk 1]
```diff
@@ -127,6 +127,9 @@ above, for each 1000 pods. Therefore, the recommended size of master host in an
 well as the minimum requirements for a master host of 2 CPU cores and 8 GB of
 RAM.
 
+When planning a multi-master environment, a minimum of three etcd hosts as well
```
Comment (tpoitras) [inline install_config/install/prerequisites.adoc]: I'm not sure on the short version "multi" here. I suggest:

When planning an environment with multiple masters...

**Comment by tpoitras** [comment]
@bfallonf LGTM aside from one nit

**Comment by bfallonf** [comment]
@tpoitras Done. Thanks. I'll merge when it's built.

**Comment by bfallonf** [comment]
[rev_history]
|link:..//install_config/install/prerequisites.html[Installing -> Prerequisites]
|Changed title to Host Recommendations and added information to section.
%

**Pull request merged**